data/issues/emissions/emission_data_calculations.py

```diff
@@ -1,21 +1,34 @@
+# pylint: disable=invalid-name
 # -*- coding: utf-8 -*-
 
 import datetime
 from dateutil.relativedelta import relativedelta
 import numpy as np
-import pandas as pd
+
 from issues.emissions.historical_data_calculations import get_n_prep_data_from_smhi
 from issues.emissions.approximated_data_calculations import calculate_approximated_historical
 from issues.emissions.trend_calculations import calculate_trend_coefficients, calculate_trend
 
+# Numbers below from
+# https://www.cemus.uu.se/wp-content/uploads/2023/12/Paris-compliant-carbon-budgets-for-Swedens-counties-.pdf
+# Found in tables on page 8 and 19
 
-BUDGET = 80000000                # C02 budget in metric tonnes
-BUDGET_YEAR = 2024               # year from which the budget applies
+# National C02 budget in metric tonnes for 50% chance of staying below 1.5 degrees
+NATIONAL_BUDGET_15 = 80000000
+# National C02 budget in metric tonnes for 50% chance of staying below 1.7 degrees
+NATIONAL_BUDGET_17 = 285113000
```

Comment:
Verfied against the table on page 19 of https://www.cemus.uu.se/wp-content/uploads/2023/12/Paris-compliant-carbon-budgets-for-Swedens-counties-.pdf :+1: